lib/utils.py

```diff
@@ -258,3 +258,10 @@ def prefix_each_line(prefix, data):
     data = data.rstrip('\n')
     lines = [(line + '\n') for line in data.split('\n')]
     return prefix + prefix.join(lines)
+
+
+def just_and_trim(s, width):
```

Comment:
pylint doesn't like such names http://pylint-messages.wikidot.com/messages:c0103 . I and Mons too)
PEP8 is on your side. So it's up to you.

Reply:
I'm completely okay with such names when its scope is short (I would use `str`, but it is a type name in python). I don't see any reason to change `with open(<…>) as f` (or `fh`) too.

Anyway, what name you propose?

Reply:
> lib/utils.py:264:4: R1705: Unnecessary "else" after "return" (no-else-return)

BTW, fixed this warning.

Reply:
Renamed to `src`.